server/sonar-web/src/main/js/apps/component-measures/drilldown/ComponentCell.js

```diff
@@ -22,17 +22,17 @@ import React from 'react';
 import QualifierIcon from '../../../components/icons-components/QualifierIcon';
 import { splitPath } from '../../../helpers/path';
 import { getComponentUrl } from '../../../helpers/urls';
-import type { ComponentEnhanced } from '../types';
+/*:: import type { ComponentEnhanced } from '../types'; */
 
-type Props = {
+/*:: type Props = {
   component: ComponentEnhanced,
   onClick: string => void
-};
+}; */
 
 export default class ComponentCell extends React.PureComponent {
-  props: Props;
+  /*:: props: Props; */
 
-  handleClick = (e: MouseEvent) => {
+  handleClick = (e /*: MouseEvent */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)